signer/build.rs

```diff
@@ -0,0 +1,4 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    tonic_build::compile_protos("../protobufs/stacks/signer/message_relay.proto")?;
+    Ok(())
```

Comment:
Where would this output the proto-based code?
(I am comparing it to Java where we specify the output)

Reply:
The output gets put in the directory specified by the `OUT_DIR` environment variable that cargo sets.

Reply:
Which is typically a directory that looks something like `target/debug/build/signer-d8ad0a07682fc412/out`